### PR TITLE
Update episode metadata

### LINKED ISSUE #4: Add content to episode Metadata
The Metadata is still a rough outline, inspired by the what and why sections of https://www.howtofair.dk and other resources available in our collaboration document. It needs to be developed further.

### PR DESCRIPTION
**Pull request by SarahAlidoost**
closes #4

### CODE REVIEW

File: _episodes/metadata.md [hunk 1]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "What are metadata?"
+- "What is an ontology (or controlled vocabulary)?"
+- "What are metadata standards?"
+- "How can I choose the appropriate metadata standard for my data?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Understand metadata"
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
- "Understand metadata and their role in FAIR data"
```

File: _episodes/metadata.md [hunk 2]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+- "Recognize different types of metadata"
+- "Apply an appropriate metadata standard for your data"
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
- "Choose an appropriate metadata standard for your data"
```

File: _episodes/metadata.md [hunk 3]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
 teaching: 0
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "What are metadata?"
+- "What is an ontology (or controlled vocabulary)?"
+- "What are metadata standards?"
+- "How can I choose the appropriate metadata standard for my data?"
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
```

File: _episodes/metadata.md [hunk 4]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+- "From a FAIR perspective, metadata are more important than your data."
+- "Metadata are preferably created according to a disciplinary standard."
+- "To be FAIR, metadata must have a findable persistent identifier."
 ---
 
+## Metadata
 
-“Metadata are data about data. Research data need metadata to become findable, accessible, interoperable and reusable - by humans and machines. “
+Metadata are data about data. In other words, metadata is the underlying definition or description of data.
+For example, author, date created, date modified and file size are examples of very basic metadata.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
For example, `author name`, `date created`, `date modified` and `file size` are examples of very basic metadata for a file.
```

File: _episodes/metadata.md [hunk 5]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
 
-Conventions for metadata (CF, CMIP, )
+Metadata can make finding and working with the data easier. Therefore, they are essential components in making your data FAIR.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
Metadata make finding and working with the data easier. Therefore, they are essential components in making your data FAIR.
```

File: _episodes/metadata.md [hunk 6]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+From a FAIR perspective, metadata are more important than your data,
+because metadata would always be openly available and they link research data and publications.
```
Comment (stefsmeets) [inline _episodes/metadata.md]: I'm not sure I agree with the last statement, as it is the persistent identifiers that link the research data and publications.

File: _episodes/metadata.md [hunk 7]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+Even if the original data are not available, it is extremely useful to track people, institutions or
+publications associated with the original research.
```
Comment (stefsmeets) [inline _episodes/metadata.md]: Here is my attempt to better organize this paragraph to better drive your point:

```suggestion
One could argue that metadata are more important than your data. 
Without metadata, the data would be just numbers. 
But, without the original data, the metadata can still be useful to track people, institutions or publications associated with the original research.
From a FAIR perspective, metadata would always be openly available.
```

File: _episodes/metadata.md [hunk 8]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
 
-Geospatial data
+Metadata can be created manually, or by automated information processing and preferably according to a disciplinary standard.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
Metadata can be created manually, or automatically generated by the software or equipment used and preferably according to a disciplinary standard.
```

File: _episodes/metadata.md [hunk 9]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
-Example convention/standards hierarchy: NetCDF > CF conventions > CMIP conventions
+> ## FAIR principles about metadata
+>
+> Let's have a look at [FAIR principles](https://www.go-fair.org/fair-principles/).
+>
+> 1. Which principles focus on metadata?
+>
+>> ## Solution
+>>
+>> 1. Because metadata are data about data,
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
>> Because metadata are data about data,
```

File: _episodes/metadata.md [hunk 10]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+>> all of the principles i.e. Findable, Accessible, Interoperable and Reusable are applied to metadata.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
>> all of the principles i.e. Findable, Accessible, Interoperable and Reusable apply to metadata.
```

File: _episodes/metadata.md [hunk 11]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+{: .challenge}
 
-Software can be useful to force you to adhere, e.g. many packages (like xarray in python) handle NetCDF files, Iris makes your data adheres to CF conventions, CMOR makes your data adhere to CMIP conventions. Units packages (cf_units?).
+## Three types of metadata
 
-What about CDO/nco, gdal?
+We focus on three main types of metadata:
 
-ESMValTool also has built in convention checkers: this ensures your data adheres to the standards.
+- **Administrative metadata** helps manage a resource or a project and indicates when and how data was created.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
- **Administrative metadata** helps manage a resource or a project and indicates when and how the data were created.
```

File: _episodes/metadata.md [hunk 12]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+For example, project/resource owner, principal investigator, project collaborators, funder, project period, permissions, etc.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
For example, the project/resource owner, principal investigator, project collaborators, funder, project period, permissions, etc.
```

File: _episodes/metadata.md [hunk 13]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+They are usually assigned to the data before you collect or create them.
 
-Other forms of data (spectral, point data, image)
+- **Descriptive or citation metadata** are information about a dataset or resource and help to discover and identify data.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
- **Descriptive or citation metadata** help to discover and identify data. A very good example of these are keywords, which are often added to data or publications with the only purpose to make them more findable (i.e. with a search engine).
```

File: _episodes/metadata.md [hunk 14]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+For example, authors, title, abstract, keywords, persistent identifier, related publications, etc.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
Other examples are the authors, title, abstract, keywords, persistent identifier, related publications, etc.
```

File: _episodes/metadata.md [hunk 15]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+
+- **Structural metadata** describes how a dataset or resource came about, but also how it is internally structured.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
- **Structural metadata** describe how a dataset or resource came about, but also how it is internally structured. They address the 'I' and 'R' in FAIR.
```

File: _episodes/metadata.md [hunk 16]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+For example, the unit of analysis, collection method, sampling procedure, sample size, categories, variables, etc.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
For example, measurement units, data collection method, sampling procedure, sample size, categories, variables, etc.
```

File: _episodes/metadata.md [hunk 17]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+Structural metadata has to be created according to best practices in a research community and will be published together with the data.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
Structural metadata have to be created according to best practices in a research community and will be published together with the data.
```

File: _episodes/metadata.md [hunk 18]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+
+> ## Keep in mind that:
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
> ## Keep your metadata up-to-date!
```

File: _episodes/metadata.md [hunk 19]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+>
+> Here are some questions about the use case you chose in the introduction of this tutorial,
+> [here]({{ page.root }}{% link _episodes/introduction.md %}).
+>
+> 1. What is the type of metadata in your use case?
+> 2. What information are described by that metadata?
+>
+{: .discussion}
+
+## Ontology (or controlled vocabulary)
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
## Ontology
```

File: _episodes/metadata.md [hunk 20]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+
+An ontology (or controlled vocabulary) is a standard definition of key concepts in your community/research team
+and focuses on how those concepts are related to one another.
+A controlled vocabulary is a set of terms that you have to pick from.
+Using an ontology helps:
+
+- others to understand the structure and content of your data,
+- make your data findable, interoperable and reusable.
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
Using an ontology:

1. helps others to understand the structure and content of your data,
2. makes your data findable, interoperable and reusable.
```

File: _episodes/metadata.md [hunk 21]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+>> ## Solution
+>>
+>> Climate and Forecast metadata ([CF conventions](http://cfconventions.org/))
+>> maintains a vocabulary specifically for specifying surface and area types.
+>> The vocabulary is available on the CF site as the
+[Area Type Table](http://cfconventions.org/Data/area-type-table/current/build/area-type-table.html).
+> {: .solution}
+{: .challenge}
+
+## Metadata standards (or conventions)
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
## Metadata standards
```

File: _episodes/metadata.md [hunk 22]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+
+A metadata standard (or convention) is a subject-specific guide to your data.
+Rules on what content must be included, what syntax must be used, or a controlled vocabulary
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
Rules on what content must be included, which syntax should be used, or a controlled vocabulary
```

File: _episodes/metadata.md [hunk 23]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+- Climate and Forecast metadata ([CF conventions](http://cfconventions.org/))
+- World Meteorological Organization Core Metadata Profile ([WMO-CMP](https://www.wmo.int/pages/prog/www/WIS/metadata_en.html))
+- Generic Earth Observation Metadata Standard ([GEOMS](https://avdc.gsfc.nasa.gov/index.php?site=1925698559))
+- Cooperative Ocean-Atmosphere Research Data Service Conventions ([COARDS](https://ferret.pmel.noaa.gov/Ferret/documentation/coards-netcdf-conventions))
+- Water Markup Language ([WaterML](https://www.ogc.org/standards/waterml))
+- Shoreline Metadata Profile of the Content Standards for Digital Geospatial Metadata ([SMP-CSDGM](https://www.fgdc.gov/standards/projects/FGDC-standards-projects/metadata/shoreline-metadata/))
+
+> ## A recognized metadata convention for climate data
+>
+> Let's do a search for the keyword `climate` in [FAIR satndards](https://fairsharing.org/standards/).
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
> Let's do a search for the keyword `climate` in [FAIR standards](https://fairsharing.org/standards/).
```

File: _episodes/metadata.md [hunk 24]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+>
+> 1. What metadata convention did you find?
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
> 1. Which metadata convention did you find?
```

File: _episodes/metadata.md [hunk 25]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+> 2. Which file format does support the implementation of the metadata?
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
> 2. Which file formats allow you to include the metadata?
```

File: _episodes/metadata.md [hunk 26]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+>
+>> ## Solution
+>>
+>> 1. Climate and Forecast metadata ([CF conventions](http://cfconventions.org/))
+>> 2. NetCDF
+>> 3. Atmospheric science, earth science, natural science, and oceanography
+> {: .solution}
+{: .challenge}
+
+> ## no/incomplete metadata standards in your research team
```
Comment (stefsmeets) [inline _episodes/metadata.md]:
```suggestion
> ## No/incomplete metadata standards in your research team
```

File: _episodes/metadata.md [hunk 27]
```diff
@@ -3,28 +3,154 @@ title: "Metadata"
...
+>
+> Assume that you want to use metadata describing processes that produce data.
+>
+> 1. The available standards in your research team are not specifically suited
+> for that purpose.
+> 2. There is no standard in your research team for that purpose.
+> How can you define a relevant metadata scheme?
```
Comment (stefsmeets) [inline _episodes/metadata.md]: I find this discussion point a bit difficult to follow. It seems like 1 and 2 are very similar.

How about:

```suggestion
> Imagine you are working in a lab, and you want to use metadata describing processes that produce data. However, the available standards in your research team are not specifically suited for that purpose. 
>
> How can you define a relevant metadata scheme?
> What would you do if there was no standard in your research team?
```

**Review by stefsmeets — CHANGES_REQUESTED** [review]
Hi @SarahAlidoost , nice work! It looks like it covers all the bases, just a few suggestions from my side.

File: _episodes/metadata.md (hunk 6 above)
Comment (SarahAlidoost) [reply _episodes/metadata.md]: Thanks. This is mentioned here https://www.howtofair.dk/how-to-fair/metadata/. However, the text is now revised.

**Comment by SarahAlidoost** [comment]
> Hi @SarahAlidoost , nice work! It looks like it covers all the bases, just a few suggestions from my side.

@stefsmeets thank you for the comments, very helpful. Your suggestions are applied.

**Review by stefsmeets — APPROVED** [review]
👍

**Pull request merged**